docs/docs/components/migrating/README.md

```diff
@@ -0,0 +1,190 @@
+
+## Key Changes
+
+**Capture user input via `props` instead of `params`** 
+
+The component model does not support `params`. You need to migrate `params` references to `props`. Unlike `params`, `props` must be explicitlly declared and defined prior to using them in code (in the old model, an input form was automatically generated when  `params` were used in code -- `params` were not explicitly declared).
+
+**Declare app `props` to use managed auth**
+
+The model for linking an app to legacy actions as well as the syntax for referencing credentials is different with Pipedream components. In the old model, apps were linked to steps in Pipedream's workflow builder UI, and credentials were referenced via the `auths` object.
+
+When using the component model, apps are defined as `props` and credentials are referenced as properties of the app. For example, to use managed auth for Github, the component `props` must contain a key (`gh` in the example below) with an app definition for the value (the app definition is an object):
+
+```javascript
+gh: {
+  type: "app",
+  app: "github"
+}
+```
+
+The component's `run()` method can then reference the credentials for Github via `this.gh.$auth.oauth_access_token`.
+
+**Develop locally and host on Github**
+Actions are no longer developed in Pipedream's UI. Develop actions locally using your preferred editor, publish to Pipedream via CLI and maintain the code in your own Github repo.
+
+**Update with a click**
+When you publish a new version of an action, you can update actions used in workflows with a click (updating legacy actions in workflows requires action steps to be deleted, re-added and re-configured).
+
+**Support for async options**
+Async options allow action authos to render a paginated drop down menu allowing users to select from vaues that are programatically generated. The most common use case is to populate the drop down based on results of an API request (e.g., to list Google Sheets in a user's drive).
+
+**Simplified discovery**
+Actions you publish are now grouped under **My Actions** when adding a step to a workflow. NOTE: this option will appear in the workflow builder *after* you publish your first action.
+
+## Getting Started
+
+Ready to develop your first component action? We recommend starting with our [quickstart guide](/components/quickstart/nodejs/actions/). Then review both our [component API reference](/components/api) and [actions published to Pipedream’s Github repo](https://github.com/pipedreamhq/pipedream/compnents).
+
+## Migration Example
+
+Let's walk through an example that migrates code for a legacy action to a Pipedream component. 
+
+### Legacy Code Example
+
+Following is the code for the legacy action to get a Github repo (Github was linked to this action via Pipedream's UI, so it's not declared in the code):
```

Comment:
Very small but technically it's "GitHub" with a capital "H". I've been bad about this in the past but I'm trying to modify it where I see it. Could be worth a Find and Replace in these docs.

Reply:
done